app/uk/gov/hmrc/alcoholdutyreturns/controllers/actions/AuthorisedAction.scala

```diff
@@ -58,6 +60,7 @@ class BaseAuthorisedAction @Inject() (
     ) {
       block(request)
     } recover { case e: AuthorisationException =>
+      logger.debug(s"Got AuthorisationException: $e")
```

Comment:
I'm assuming we want this one at debug level since it could spam the log in a DOS (however we still log other stuff anyway). Comments?

Reply:
Yeah debug sounds good